test/lib/mayaUsd/render/pxrUsdMayaGL/testProxyShapeLiveSurface.py

```diff
@@ -69,8 +69,7 @@ def setUp(self):
         cmds.refresh()
 
         # Get the viewport widget.
-        self._view = OMUI.M3dView()
-        OMUI.M3dView.getM3dViewFromModelPanel(self._panel, self._view)
+        self._view = OMUI.M3dView().active3dView()
```

Comment:
What does this change? The code seems to be setting the panel to a persp view and then getting the viewport from it. Now you are using the active viewport. Is that still the one with persp camera?

Reply:
Yes, there is only on viewport open at that point. The old code would not return the correct panel when I ran it locally.